Marlin/language_de.h

```diff
@@ -153,6 +159,12 @@
 #define MSG_ERR_MINTEMP                     LCD_STR_THERMOMETER " UNTERSCHRITTEN"
 #define MSG_ERR_MAXTEMP_BED                 "BETT " LCD_STR_THERMOMETER " ÜBERSCHRITTEN"
 #define MSG_ERR_MINTEMP_BED                 "BETT " LCD_STR_THERMOMETER " UNTERSCHRITTEN"
+#define MSG_ERR_Z_HOMING                    "G28 Z Forbidden"
```

Comment:
You are right. This is not clean. But saves the space for a second string.
However. Hosts do not react on that, when printed with 'echo'. Even when printed with 'error' the host should react with stopping to send on __ every__ unknown error.